### DISCUSSION TITLE
ERROR: Failed opening data/assets/demo.tmx

### DISCUSSION THREAD

**Discussion by vinay** [fallahn/tmxlite#147] [Q&A]
Im trying to use this library in xcode, and its failing while trying to load the sample provided. Im able to load other assets from the data folder referenced in the error message.

Any help would be great.

**Comment by fallahn** [comment]
Not entirely sure why - the path should be `assets/demo.tmx` not `data/assets/demo.tmx` so you may have a mistake there (see [this](https://github.com/fallahn/tmxlite/blob/master/SFMLExample/src/main.cpp#L40) line). XCode also expects the assets directory to be in the Debug or Release output directories so you can either copy it there or create a symlink to the repository directory:

    ln -s ~/repos/tmxlite/SFMLExample/assets/ ~/my/build/directory/Debug

see attached for example
![untitled-1](https://user-images.githubusercontent.com/5619252/52047772-a43bdf80-2541-11e9-89ee-88911927d3a5.png)

**Comment by vinay** [comment]
i guess it has something to do with Xcode manages file paths. I tried a much simpler example to just load a FILE from a path, and it fails too. Tried giving it both the relative and absolute paths and it still doesnt work. 

I got the tileset to load on windows by the way.
